server/service/src/invoice_line/stock_in_line/update/generate.rs

```diff
@@ -0,0 +1,149 @@
+use crate::{
```

Comment:
are the `stock_in_line` and `stock_out_line` just for returns?
I was confused bc I thought we'd probably have stock in/out already. Perhaps add 'return' in the name or pop in a new folder? some sort of distinction would be helpful (for me at least!)

Reply:
We have stock_out, but not stock_in yet! The plan was to create this abstraction for returns, and at a later date, refactor inbound shipments to also use the stock_in abstraction!